tests/test.py

```diff
@@ -61,6 +65,31 @@ def test_open_device_more_than_once(self):
         with self.assertRaises(DeviceAlreadyMountedError):
             self.device.open()
 
+    def test_id_overflow(self):
+        if self.use_simulator:
+            self.skipTest("For real device only")
+
+        d = SupernovaDevice(start_id=655535)
```

Comment:
I don't know if intended but here you started with 655535 and maybe you wanted 65535?